### PR TITLE
upgraded infrastructure for new chlu-ipfs

### PR DESCRIPTION
**Pull request by fazo96**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "chlu-did-service": "ChluNetwork/chlu-did-service",
+    "chlu-did-service": "ChluNetwork/chlu-did-service#fix-chlu-did",
```
Comment (kulpreet) [inline package.json]: Should we move this branch dependency out?

**Review by kulpreet — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (fazo96) [reply package.json]: @kulpreet yes but we got https://github.com/ChluNetwork/chlu-did-service/pull/1 to merge

Comment (kulpreet) [reply package.json]: Just merged that, so this should be unblocked. I can just edit it here and merge, might save us time.

Comment (fazo96) [reply package.json]: no, it needs a `yarn` and `yarn upgrade chlu-did-service` run to fix the `yarn.lock`. I just pushed a new commit

**Pull request merged**